src/views/Playlist/CuratedPlaylistDetails.vue

```diff
@@ -14,6 +17,20 @@ const playlistId = Number(props.playlistId);
 const playlist: Ref<PlaylistModel> = ref({});
 const tracks: Ref<TrackModel[]> = ref([]);
 
+const { setCurrentSong } = inject(MediaPlaylistInjectionKey)!;
+// TODO: Please move this into a plugin or something the-like which configures the authorize-url function.
+let token: string = "";
+(async () => {
+  token =
```

Comment:
I would have expected that setCurrentSong takes care of that

Reply:
Debatable, but thanks! I'll move it into the player. It seems to make it a lot easier when dealing with playlists.

Reply:
I think this is the only thing preventing this from being merged

Reply:
Should be fixed now.